### PR TITLE
[no ticket][risk=no] Puppeteer fix sidebar code issues

### PR DESCRIPTION
**Pull request by aweng98**
`help-sidebar.ts` is separated into two smaller classes, `review-criteria-sidebar.ts` and `review-conceptset-sidebar.ts` because they're two different sidebars.
Both sidebar classes extends `BaseHelpSidebar.ts` abstract class.
Removed unused and duplicate functions.

### CODE REVIEW

File: e2e/app/element/base-element.ts [hunk 1]
```diff
@@ -146,21 +146,25 @@ export default class BaseElement extends Container {
...
             const box = await element.boundingBox();
             const x = box.x + (box.width / 2);
             const y = box.y + (box.height / 2);
+            if (previousX !== undefined && previousY !== undefined) {
+              // tslint:disable:triple-equals
+              if (parseFloat(previousX.toFixed(7)) == parseFloat(x.toFixed(7))
+                && parseFloat(previousY.toFixed(7)) == parseFloat(y.toFixed(7))) {
+                break;
+              }
+            }
```
Comment (aweng98) [inline e2e/app/element/base-element.ts:164]: unrelated fix. comparing two float numbers was not working.

**Review by aweng98 — COMMENTED** [review]
@dolbeew Hi Will, I'm making some changes which are affecting Cohort/Dataset tests. Can you help with review? Thanks

**Review by dolbeew — APPROVED** [review]
Thanks, LGTM.
